packages/cli/commands/cms/convertFields.js

```diff
@@ -27,7 +27,10 @@ const invalidPath = src => {
 
 exports.handler = async options => {
   const src = path.resolve(getCwd(), options.src);
-  const projectRoot = path.dirname(getThemeJSONPath(src));
+  const themeJSONPath = getThemeJSONPath(src);
+  const projectRoot = themeJSONPath
+    ? path.dirname(themeJSONPath)
+    : path.dirname(getCwd());
```

Comment:
`projectRoot` shouldn't be `undefined` because we'll fail to get the correct relative path in Windows